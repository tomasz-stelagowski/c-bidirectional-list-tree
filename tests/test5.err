NODES: 2
NODES: 3
NODES: 4
NODES: 5
NODES: 6
NODES: 7
NODES: 8
NODES: 9
NODES: 10
NODES: 11
NODES: 10
NODES: 10
NODES: 10
NODES: 10
NODES: 11
NODES: 10
NODES: 11
NODES: 10
NODES: 10
NODES: 10
NODES: 10
NODES: 11
NODES: 10
NODES: 11
NODES: 10
NODES: 10
NODES: 10
NODES: 10
NODES: 11
NODES: 10
NODES: 11
NODES: 9
NODES: 10
NODES: 10
NODES: 10
NODES: 11
NODES: 10
NODES: 11
NODES: 10
NODES: 10
NODES: 10
NODES: 10
NODES: 11
NODES: 10
NODES: 11
NODES: 10
NODES: 10
NODES: 10
NODES: 10
NODES: 11
NODES: 11
NODES: 11
NODES: 11
NODES: 11
NODES: 11
NODES: 11
