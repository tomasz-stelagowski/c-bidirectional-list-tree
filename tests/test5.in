ADD_NODE 0
SPLIT_NODE 0 1
ADD_NODE 2
SPLIT_NODE 0 2
ADD_NODE 1
SPLIT_NODE 0 1
ADD_NODE 1
SPLIT_NODE 0 6
ADD_NODE 3
SPLIT_NODE 3 9
DELETE_SUBTREE 5
RIGHTMOST_CHILD 1
RIGHTMOST_CHILD 6
RIGHTMOST_CHILD 2
ADD_NODE 6
DELETE_NODE 2
SPLIT_NODE 1 7
DELETE_SUBTREE 12
RIGHTMOST_CHILD 1
RIGHTMOST_CHILD 6
RIGHTMOST_CHILD 11
ADD_NODE 7
DELETE_NODE 10
SPLIT_NODE 7 13
DELETE_SUBTREE 11
RIGHTMOST_CHILD 14
RIGHTMOST_CHILD 7
RIGHTMOST_CHILD 8
ADD_NODE 13
DELETE_NODE 6
SPLIT_NODE 7 13
DELETE_SUBTREE 13
ADD_NODE 4
RIGHTMOST_CHILD 14
RIGHTMOST_CHILD 7
ADD_NODE 8
DELETE_NODE 1
SPLIT_NODE 8 18
DELETE_SUBTREE 19
RIGHTMOST_CHILD 18
RIGHTMOST_CHILD 9
RIGHTMOST_CHILD 8
ADD_NODE 17
DELETE_NODE 9
SPLIT_NODE 16 14
DELETE_SUBTREE 21
RIGHTMOST_CHILD 16
RIGHTMOST_CHILD 0
RIGHTMOST_CHILD 20
ADD_NODE 4
RIGHTMOST_CHILD 4
RIGHTMOST_CHILD 17
RIGHTMOST_CHILD 0
RIGHTMOST_CHILD 7
RIGHTMOST_CHILD 20
RIGHTMOST_CHILD 22
