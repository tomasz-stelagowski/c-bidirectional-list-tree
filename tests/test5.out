OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
7
4
3
OK
OK
OK
OK
7
11
-1
OK
OK
OK
OK
-1
14
-1
OK
OK
OK
OK
OK
-1
16
OK
OK
OK
OK
-1
-1
18
OK
OK
OK
OK
14
8
-1
OK
22
20
8
16
-1
-1
